packages/frontend/src/routes/signIn/index.tsx

```diff
@@ -136,27 +146,29 @@ function SignIn() {
               <button
                 type="button"
                 onClick={handleGoogleSignIn}
-                className="flex items-center justify-center w-full px-4 py-2 text-white bg-blue-600 rounded-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500"
+                disabled={loading}
+                className="flex items-center justify-center w-full px-4 py-2 text-white bg-blue-600 rounded-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500 disabled:opacity-50"
               >
-                <FaGoogle className="mr-2" />
+                {oauthMethod === "google" && loading ? (
+                  <LoadingIcon className="w-8 h-8 animate-spin text-primary-white" />
```

Comment:
Can we maintain the same loader used across the app?

Reply:
Where is the Loader's name?